source/Slice/Family.lagda

```diff
@@ -71,3 +72,28 @@ resize-family : {A : 𝓤  ̇}
 resize-family S (A₀ , s , e) = A₀ , (λ x → S [ s x ])
 
 \end{code}
+
+We will now add mechanisms for turning subsets into families for increased
```

Comment:
Somehow it reads better to me to start this sentence with "For increased readability".